account_move_line_accounting_description/reports/invoice.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint : disable=dangerous-qweb-replace-wo-priority -->
```

Comment:
Instead of replacing `t-field="line.name"`, please make it `invisible` and then add your field after it.